modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/resource/v1_0/UserResourceImpl.java

```diff
@@ -57,13 +58,13 @@ public Object getV2UserById(String id) throws Exception {
 	}
 
 	@Override
-	public Object getV2Users(Integer count, Integer startIndex)
+	public Object getV2Users(Integer count, Integer startIndex, Filter filter)
 		throws Exception {
 
 		return _buildResponse(
 			_userResourceManager.listWithGET(
-				_userManager, null, startIndex, count, null, null, null, null,
-				null));
+				_userManager, contextHttpServletRequest.getParameter("filter"),
```

Comment:
@jgarcialr can we use the ParamUtil.getString here?

Reply:
yes, we need to add a default value of null in order to charon work as expected if no filter is provided